editor/src/components/inspector/sections/component-section/variables-in-scope-utils.ts

```diff
@@ -17,6 +17,7 @@ function valuesFromObject(
   value: object | null,
   depth: number,
   displayName: string,
+  advanced: boolean,
```

Comment:
I think advanced is a little hard to understand, makes me feel like this is a very advanced property, only for pro use :) 

what about `matchingType` or `typeMatches` or `matchesProp` or `highlighted` or `valueMatchesPropType`?

Reply:
fair point, I'll go with `valueMatchesPropType`